docs/data/pages.ts

```diff
@@ -1,6 +1,11 @@
 import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
 
 const pages: MuiPage[] = [
+  {
+    pathname: '/x/whats-new',
+    title: "✨ What's new in v6? ✨",
```

Comment:
```suggestion
    title: "What's new in v6? ✨",
```

To keep it aligned with the rest of the items in the nav bar?

Reply:
That was my first try, but tbh, I think it looked better and called more attention this way. 
We can merge it like this and evaluate this change with @gerdadesign on the follow-up PRs.

Reply:
Btw, I'm not sure that the eye icon is the best one we can use either.

Reply:
Agreed. We use the eye icon for "How-to guides" throughout Core.
<a href="https://mui.com/material-ui/guides/api/"><img src="https://user-images.githubusercontent.com/106991024/217629734-f0a5eb8d-2008-4faf-bd81-bf9fc5d1a6cd.png" alt="MaterialUI_nav_guides" width="200"></a>

For this, I found a couple icons that could work well. For the text, I don't mind the ✨ on either side to draw more attention to it, but it's not as visible in light mode. There is also the 🆕 emoji, but could be redundant with the copy and icon. 
If we want to prioritize consistency, only keeping the stars at the end works. If we want to bring more emphasis to this, we could do `text-transform: uppercase` (but keep sentence case in the main page title). We could also style this differently to bring even more emphasis to it, but it depends just how different we want it to be! 

Here's some preview images: 

|  | Light | Dark | 
| ------------- | ------------- | ------------- |
| [NewReleases icon](https://mui.com/material-ui/material-icons/?query=new&selected=NewReleases)  | ![NewReleases_light](https://user-images.githubusercontent.com/106991024/217630801-d8b24997-1484-4bec-a328-d586dcca59a5.png) | ![NewReleases_dark](https://user-images.githubusercontent.com/106991024/217630798-3bcd0e15-c80b-4740-946b-6dd3babadfb1.png) | 
| [FiberNew icon](https://mui.com/material-ui/material-icons/?query=new&selected=FiberNew)  | ![FiberNew_light](https://user-images.githubusercontent.com/106991024/217630800-7da5c6d1-fff2-46c1-a202-0a44a2727850.png) | ![FiberNew_dark](https://user-images.githubusercontent.com/106991024/217630799-8d93b3a6-18b5-403a-957d-27963b3534af.png) |
| 🆕 emoji instead of an icon to differentiate | ![NewEmoji_reg_notselected_light](https://user-images.githubusercontent.com/106991024/217635758-48feb240-b85b-411f-a9b6-93877129b46f.png) |  ![NewEmoji_reg_notselected_dark](https://user-images.githubusercontent.com/106991024/217635808-66af2fa8-6c1e-45d6-a55a-e57ac3e51eab.png) |

Reply:
Thanks @gerdadesign 
I like the first one with the NewReleasesIcon!
I'll submit a PR to update it